internal/arista/arista.go

```diff
@@ -77,3 +79,28 @@ func (n *Namer) Fabric(index int) (string, error) {
 	}
 	return fmt.Sprintf("Fabric%d", index+1), nil
 }
+
+// Port is an Arista implementation of namer.Port.
+func (n *Namer) Port(pp *namer.PortParams) (string, error) {
+	var nameBuilder strings.Builder
```

Comment:
nil guard? Same for all vendor impls.

Reply:
No for the vendor implementations. The burden on the vendor implementations should be minimal. They are internal and can always assume they are called with valid arguments.